[amazonechocontrol] AccountHandler: refresh data fails with unexpected error

I'm receiving the following error in the log after a restart of OH and several times throughoiut the day. I'm failr confident that this began happening after enabling Severe Weather Alerts, which was [added in February](https://www.amazon.com/b?node=21100773011).
```
2020-04-16 01:12:25.426 [ERROR] [org.openhab.binding.amazonechocontrol.internal.handler.AccountHandler] - refresh data fails with unexpected error
java.time.format.DateTimeParseException: Text '2020-04-16Tnull-04:00' could not be parsed at index 11
        at java.time.format.DateTimeFormatter.parseResolved0(DateTimeFormatter.java:1949) ~[?:1.8.0_242]
        at java.time.format.DateTimeFormatter.parse(DateTimeFormatter.java:1851) ~[?:1.8.0_242]
        at java.time.ZonedDateTime.parse(ZonedDateTime.java:597) ~[?:1.8.0_242]
        at java.time.ZonedDateTime.parse(ZonedDateTime.java:582) ~[?:1.8.0_242]
        at org.openhab.binding.amazonechocontrol.internal.handler.EchoHandler.updateNotifications(EchoHandler.java:1308) ~[?:?]
        at org.openhab.binding.amazonechocontrol.internal.handler.AccountHandler.refreshNotifications(AccountHandler.java:437) ~[?:?]
        at org.openhab.binding.amazonechocontrol.internal.handler.AccountHandler.refreshData(AccountHandler.java:538) ~[?:?]
        at org.openhab.binding.amazonechocontrol.internal.handler.AccountHandler.checkData(AccountHandler.java:406) ~[?:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_242]
        at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) [?:1.8.0_242]
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_242]
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) [?:1.8.0_242]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_242]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_242]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_242]
```
![image](https://user-images.githubusercontent.com/21967853/79419686-49937580-7f85-11ea-9287-dcc4a36b30b7.png)


## Your Environment
* Version used: OH 2.5.4 S86, amazonechocontrol 2.5.4.202004150344

--- a/bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/handler/EchoHandler.java
+++ b/bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/handler/EchoHandler.java
@@ -19,6 +19,7 @@ import java.net.URISyntaxException;
 import java.time.Instant;
 import java.time.ZoneId;
 import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
 import java.time.temporal.ChronoUnit;
 import java.util.*;
 import java.util.concurrent.ScheduledFuture;
@@ -1278,8 +1279,11 @@ public class EchoHandler extends BaseThingHandler implements IEchoThingHandler {
                 if ("ON".equals(notification.status)) {
                     if ("Reminder".equals(notification.type)) {
                         String offset = ZoneId.systemDefault().getRules().getOffset(Instant.now()).toString();
-                        ZonedDateTime alarmTime = ZonedDateTime
-                                .parse(notification.originalDate + "T" + notification.originalTime + offset);
+                        String date = notification.originalDate != null ? notification.originalDate
+                                : ZonedDateTime.now().toLocalDate().format(DateTimeFormatter.ISO_LOCAL_DATE);
+                        String time = notification.originalTime != null ? notification.originalTime : "00:00:00";
+                        ZonedDateTime alarmTime = ZonedDateTime.parse(date + "T" + time + offset,
+                                DateTimeFormatter.ISO_DATE_TIME);
                         String recurringPattern = notification.recurringPattern;
                         if (recurringPattern != null && !recurringPattern.isBlank() && alarmTime.isBefore(now)) {
                             continue; // Ignore recurring entry if alarm time is before now

